web/src/js/ducks/flows.ts

```diff
@@ -100,6 +101,14 @@ export default function reducer(state: FlowsState = defaultState, action): Flows
                 selected: action.flowIds
             }
 
+        case REQUEST_ACTION:
+        case REQUEST_ACTION_FILTERED:
+            let args = action.type == REQUEST_ACTION_FILTERED ? ('?filter=' + state.filter) : ''
+            if (process.env.NODE_ENV !== 'test') {
+                window.location.href = '/flows/dump' + args
+            }
+            return state
+
```

Comment:
On closer inspection, I don't quite understand why we want to deal with dispatch/redux here at all. Can we just put the `window.location.href` part into `FileMenu.tsx`?

Reply:
It was nearly that convoluted before, but you are right. I just didn't know how to access the relevant filter state from the component, but the sibling files showed me.